tests/modules/entrezdirect/xtract/main.nf

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ENTREZDIRECT_ESUMMARY } from '../../../../modules/entrezdirect/esummary/main.nf'
+include { ENTREZDIRECT_XTRACT   } from '../../../../modules/entrezdirect/xtract/main.nf'
+
+//
+// Test with Assembly database
+//
+workflow test_entrezdirect_xtract_assembly {
+
+    input = [
+        [ id:'test_assembly' ], // meta map
+        uid = '191021',
+        []
+    ]
+    database = 'assembly'
+
+    pattern_in = 'DocumentSummary'
+    element_in = 'SpeciesName BioprojectAccn FtpPath_GenBank'
+    delim = ","
+
+    ENTREZDIRECT_ESUMMARY ( input, database )
+    ENTREZDIRECT_XTRACT ( ENTREZDIRECT_ESUMMARY.out.xml_esummary, pattern_in, element_in, delim )
```

Comment:
@alxndrdiaz . I know it was accepted as `xml_esummary`, but I think the output in `entrezdirect/esummary` should be simply called `xml` in the module and here.

Reply:
Done , here is the PR: [renamed output in entrezdirect/esearch: xml #1928](https://github.com/nf-core/modules/pull/1928).